__tests__/bin-readme.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+var path = require('path'),
+  os = require('os'),
+  exec = require('child_process').exec,
+  tmp = require('tmp'),
+  fs = require('fs-extra');
+
+function documentation(args, options, parseJSON) {
+  return new Promise((resolve, reject) => {
+    if (!options.cwd) {
+      options.cwd = __dirname;
+    }
+
+    options.maxBuffer = 1024 * 1024;
+
+    args.unshift(
+      'node ' + path.join(__dirname, '..', 'bin', 'documentation.js')
+    );
+
+    exec(args.join(' '), options, (err, res) => {
+      resolve(res);
+    });
+  });
+}
+
+var UPDATE = !!process.env.UPDATE;
```

Comment:
Do we still need this?

We (I) can change this in a follow up if it works now. I don't want to make this PR any larger.